lineapy/plugins/kubeflow_pipeline_writer.py

```diff
@@ -175,8 +175,9 @@ def get_rendered_task_definitions(
                 loading_blocks=loading_blocks,
                 pre_call_block="",
                 call_block=task_def.call_block,
-                post_call_block="",
+                post_call_block=task_def.post_call_block,
```

Comment:
setting up potential conflict with lazar

Reply:
good catch, must have slipped in when i was demo-ing to him, will revert.